### DISCUSSION TITLE
Upgrading from 9 -> 10 old pass keys do not work

### DISCUSSION THREAD

**Discussion by lukepolo** [MasterKale/SimpleWebAuthn#596] [Troubleshooting]
Is it expected that old passkeys no longer work when upgrading from 9 -> 10?

**Comment by MasterKale** [comment]
Hello @lukepolo, of course that's not expected. What's going wrong?

**Reply by lukepolo** [reply]
im going to dig deeper into this, im going to assume its my issue. Will post on findings.

**Reply by lukepolo** [reply]
note: new passkeys work. 😭

**Reply by lukepolo** [reply]
Ok so i did find it 

```
 logger.info({
      pre:request.body.response.userHandle,
      post: isoBase64URL.toUTF8String(request.body.response.userHandle),
    })

    try {
      const devices = await this.auth.getUserDevices(
        request.body.response.userHandle,
      );

```



My user handels from old pass keys dont need to be adjusted while my newer passkeys do.  It is probably because of my frontend using an older version vs my backend, Ill update the frontend here soon

**Reply by lukepolo** [reply]
upgraded to the latest UI .Still same issue. Any recommendations ?
